python/lsst/ts/standardscripts/maintel/m2/check_actuators.py

```diff
@@ -0,0 +1,262 @@
+                description: Actuators to run the bump test.
+                oneOf:
+                  - type: array
+                    items:
+                      type: number
+                      enum: [{m2_actuator_ids_str}]
+                    minItems: 1
+                    uniqueItems: true
+                    additionalItems: false
+                  - type: string
+                    enum: ["all"]
+                default: "all"
+            ignore_actuators:
+                description: Actuators to ignore during the bump test.
+                type: array
+                items:
+                    type: number
+                    enum: [{m2_actuator_ids_str}]
+                default: []
+        additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super().get_schema()
+
+        for properties in base_schema_dict["properties"]:
+            schema_dict["properties"][properties] = base_schema_dict["properties"][
+                properties
+            ]
+
+        return schema_dict
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration
+        """
+        # Get list of M2 hardpoints from mtcs
+        # this should be async await?
+        self.hardpoint_ids = await self.mtcs.get_m2_hardpoints()
+
+        # Getting list of actuator ids
+        num_axial_actuator = NUM_ACTUATOR - NUM_TANGENT_LINK
+        self.axial_actuator_ids = list(
+            [
+                actuator_id
+                for actuator_id in np.arange(num_axial_actuator)
+                if actuator_id + 1 not in self.hardpoint_ids
+            ]
+        )
+
+        # Actuators that will be effectively tested, filtering out hardpoints
+        # (which are 1 indexed, csc is 0 indexed)
+        if config.actuators == "all":
+            self.actuators_to_test = self.axial_actuator_ids
+        else:
+            # check that no actuators are hardpoints
+            for actuator_id in config.actuators:
+                if actuator_id + 1 in self.hardpoint_ids:
+                    raise ValueError(
+                        f"Cannot run bump test on actuator {actuator_id},"
+                        "it is currently configured as a hardpoint"
+                    )
+                if actuator_id + 1 not in self.axial_actuator_ids:
+                    raise ValueError(
+                        f"Cannot run bump test on actuator {actuator_id}, it is not a valid axial actuator."
+                    )
+
+            self.actuators_to_test = config.actuators
+
+        if config.ignore_actuators:
+            self.actuators_to_test = [
+                actuator_id
+                for actuator_id in self.actuators_to_test
+                if actuator_id not in config.ignore_actuators
+            ]
+        if hasattr(config, "period"):
```

Comment:
Neither `period` nor `force` are in the configuration above.

Reply:
Added, check that this was done correctly.